src/idmap.rs

```diff
@@ -16,7 +16,7 @@ use crate::{
 use core::ptr::NonNull;
 
 /// Identity mapping, where every virtual address is either unmapped or mapped to the identical IPA.
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
We may as well continue to implement `Copy` and `Clone` here seeing as `IdTranslation` is cheap to copy, even though it's not required by the interface anymore.